st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -67,18 +63,15 @@ def _match_tuple_to_dict(self, match):
             'representation': match[2]
         }
 
-    @request_user_has_permission(permission_type=PermissionType.ACTION_ALIAS_LIST)
-    @jsexpose()
     def get_all(self, **kwargs):
         return super(ActionAliasController, self)._get_all(**kwargs)
 
-    @request_user_has_resource_db_permission(permission_type=PermissionType.ACTION_ALIAS_VIEW)
-    @jsexpose(arg_types=[str])
-    def get_one(self, ref_or_id):
-        return super(ActionAliasController, self)._get_one(ref_or_id)
+    def get_one(self, ref_or_id, requester_user):
```

Comment:
Well, that's simple. Every controller should have rbac definition of some kind. Let's make it a part of TestApp base class.